[14.0][IMP] Kickstarter-page: update CONTRIBUTING.md (#67)
OCA/oca-custom: CONTRIBUTING.md

> **Note**: in this repository we allow unreleased dependencies.

#### Deployment

Ask administrator to deploy the given commit.

### How-Tos

Here we focus on how to do it. It's a suggested way to work but feel free to use your own way.

#### Setup developer environment

Requirements:

- Postgresql

> - [uv](https://docs.astral.sh/uv/), to be installed with `curl -LsSf https://astral.sh/uv/install.sh | sh`

Reviewer: There are several different methods to install uv, so I don't think this document is the place to mention one specifically.

Author: The intention here was to provide a self-standing guide for a developer to be ready to contribute, even with low-knowledge of tooling such as uv, the reasong why to provide a ready-to-use command line into the guide.
What about:
```suggestion
- **uv**: several methods exist to install it, one can be `curl -LsSf https://astral.sh/uv/install.sh | sh`
```

Reviewer: Yeah, it's probably just me who likes reading documentation :)

Your proposal sounds good.

nit: `several methods *exist* to install it`

- Some dependencies to be able to build some python packages: `libpq-dev`,
  `build-essential`, TODO

*** TODO ****

*** TODO ****

*** TODO ****

> 

Reviewer: Why adding these TODO's?

Author: Hello, I am actually not sure anymore.
@petrus-v @thomaspaulb do you see any other dependencies to be listed here? Even though if there were any, they would be downloaded automatically while running the `uv sync`, right?
Else we should indeed removes those TODO.

- wkhtmltopdf

Run following commands to prepare a python virtual environment with the correct python version (which uv will download for you if necessary) and install the required dependencies:

```bash

> git clone git@github.com:OCA/oca-custom -b 14.0 --single-branch --depth=1

Reviewer: ```suggestion
git clone git@github.com:OCA/oca-custom -b 14.0
```

Run following commands to prepare a python virtual environment with the correct python version (which uv will download for you if necessary) and install the required dependencies:

```bash

> git clone git@github.com:OCA/oca-custom -b 14.0

Reviewer: ```suggestion
git clone git@github.com:OCA/oca-custom --branch 14.0
```

Since we are teaching here, let's use full option names that are easier to understand.
